ui/src/reusable_ui/types/index.ts

```diff
@@ -43,3 +50,71 @@ export enum Greys {
   Ghost = '#fafafc',
   White = '#ffffff',
 }
+
+export enum IconFont {
+  Alerts = 'alerts',
+  AuthZero = 'authzero',
+  BarChart = 'bar-chart',
+  Capacitor = 'capacitor2',
+  CaretDown = 'caret-down',
+  CaretLeft = 'caret-left',
+  CaretRight = 'caret-right',
+  CaretUp = 'caret-up',
+  Checkmark = 'checkmark',
+  Circle = 'circle',
+  Clock = 'clock',
+  CogOutline = 'cog-outline',
+  CogThick = 'cog-thick',
+  Collapse = 'collapse',
+  CrownOutline = 'crown-outline',
+  CrownSolid = 'crown2',
+  Cube = 'cube',
+  Cubouniform = 'cubo-uniform',
+  DashF = 'dash-f',
+  DashH = 'dash-h',
+  DashJ = 'dash-j',
+  Disks = 'disks',
+  Download = 'download',
+  Duplicate = 'duplicate',
+  ExpandA = 'expand-a',
+  ExpandB = 'expand-b',
+  Export = 'export',
+  Eye = 'eye',
+  EyeClosed = 'eye-closed',
+  EyeOpen = 'eye-open',
+  GraphLine = 'graphline2',
+  Group = 'group',
+  Heroku = 'heroku',
+  herokusimple = '',
```

Comment:
any reason this is lowercase? seems like it'd be `HerokuSimple`

Reply:
good catch